stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.6.4
+version: 2.6.5
```

Comment:
As you are adding new features this should be a minor version not a patch.

Reply:
:+1: https://github.com/helm/charts/pull/17494/commits/8c9dd3bc8861990633e73bd229737583f98eaf8f